src/main/scala/example/TaskInitialization.scala

```diff
@@ -0,0 +1,21 @@
+package example
+
+import org.apache.spark.sql.SparkSession
+
+object TaskInitialization {
+  // CREATE SPARK SESSION
+
+  def createSparkSession: SparkSession = SparkSession.builder
+    .appName("Spark Scala Task")
+    .getOrCreate()
+
+  def createLocalSparkSession: SparkSession = {
+    val ss = SparkSession.builder
+      .appName("Spark Scala Task")
+      .master("local[*]")
+      .getOrCreate()
+    ss.conf.set("spark.sql.autoBroadcastJoinThreshold", -1)
```

Comment:
what this exactly mean?

Reply:
I disabled broadcast joins to see how the execution plans and computations would look like for larger datasets. For smaller ones like in unit tests it would be always or almost always broadcast joins.

Reply:
Unit tests should be as fast as possible, so it does not make sense to do it. But if we used the approach you proposed (with reading data from tables), then at least in theory it looks like doable.